### PR TITLE
next step for the big change, product, cart and order need to be suit…

### PR DESCRIPTION
**Pull request by eilonasher**
…e for the last version we talked about:

account need to be: name, uid(unique value), cart, static currentAccount
the other departments will be changed in the fit branches

### CODE REVIEW

File: app/src/main/java/com/example/sabonit/Account.java [hunk 1]
```diff
@@ -13,29 +13,17 @@
...
-
 
     /* ********* Attributes: ********* */
-    // Given as an address in the next pattern: street and building number, city ("Ramban 30, Jerusalem")
-    private String location;
-    // User's name, may can be extracted from the google acount
+    // User's name, may can be extracted from the google account
     private String name;
-    // User's phone number, may can be extracted from the google acount
-    private String phoneNumber;
```
Comment (liormoalem) [inline app/src/main/java/com/example/sabonit/Account.java:29]: did you make sure you removed it from every place in the code?

File: app/src/main/java/com/example/sabonit/UserLoginActivity.java [hunk 2]
```diff
@@ -93,11 +91,6 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
                     // FirebaseUser.getIdToken() instead.
                     uid = user.getUid();
                     updateAccount();
-//                    if (! isAccountExist)
```
Comment (liormoalem) [inline app/src/main/java/com/example/sabonit/UserLoginActivity.java:96]: why did you remove it? 
did you check that everything works properly?

File: app/src/main/java/com/example/sabonit/Account.java (hunk 1 above)
Comment (eilonasher) [reply app/src/main/java/com/example/sabonit/Account.java:29]: yes, the application runs fine, but it create problems we will fix now

File: app/src/main/java/com/example/sabonit/Account.java [hunk 3]
```diff
@@ -13,29 +13,17 @@
...
     // User's cart, empty when initialized
     private Cart cart;
+    //User's UID of the google account, unique value
+    private String UID;
     // The account of the user that is logged in
     private static Account currentAccount;
```
Comment (yairelboher) [inline app/src/main/java/com/example/sabonit/Account.java:25]: i wonder if this solution for the access to currentAccount is a good one...
static variables are related to the class itself, not to the instant.
did you think about the option of transfer this data through the intant (i mean, between the activities)?

File: app/src/main/java/com/example/sabonit/UserLoginActivity.java (hunk 2 above)
Comment (eilonasher) [reply app/src/main/java/com/example/sabonit/UserLoginActivity.java:96]: i think so, checked it on 2 devices and need to notice if it override new account, in my checks - it didnt

**Review by yairelboher — COMMENTED** [review]
we have no time, so if you are convinced that this solution (static variable) is good - merge it.
explain shortly if you can

File: app/src/main/java/com/example/sabonit/Account.java (hunk 3 above)
Comment (eilonasher) [reply app/src/main/java/com/example/sabonit/Account.java:25]: In my opinion the solutions are equal. ma gam that I talked with Moshe and Alex and there is no preference for one of them.
just notice that this was the original solution of gal, not mine, i just didnt change it now
